### PR TITLE
Upgrade Android and IOS version

### PR DESCRIPTION
**Pull request by mansi-square**
Upgrade Android version to 1.6.6
Upgrade IOS version to 1.6.3

## Related issues

Fix #

## Changelog

* message

### CODE REVIEW

File: example/lib/widgets/order_sheet.dart [hunk 1]
```diff
@@ -147,6 +147,8 @@ class OrderSheet extends StatelessWidget {
           CookieButton(
             text: "Buyer Verification",
             onPressed: () {
+
+              Scaffold.of(context).showBottomSheet((context) =>Container());
```
Comment (plinio-square) [inline example/lib/widgets/order_sheet.dart:151]: is this change related to the SDK upgrades?

File: example/lib/widgets/modal_bottom_sheet.dart [hunk 2]
```diff
@@ -237,8 +237,8 @@ Future<T?> showModalBottomSheet<T>({
       ));
 }
 
-PersistentBottomSheetController<T> showMyBottomSheet<T>({
+PersistentBottomSheetController showMyBottomSheet<T>({
```
Comment (plinio-square) [inline example/lib/widgets/modal_bottom_sheet.dart:240]: is this change related to the SDK upgrades?

Comment (mansi-square) [reply example/lib/widgets/modal_bottom_sheet.dart:240]: Yes,

File: example/lib/widgets/order_sheet.dart (hunk 1 above)
Comment (mansi-square) [reply example/lib/widgets/order_sheet.dart:151]: No, I've removed it's not related for SDK upgrade and i've upgraded PR.

File: example/lib/widgets/modal_bottom_sheet.dart (hunk 2 above)
Comment (mansi-square) [reply example/lib/widgets/modal_bottom_sheet.dart:240]: Yes, With latest flutter version the <T> datatype is not support as there are some changes in Dart code.

File: example/lib/widgets/order_sheet.dart [hunk 3]
```diff
@@ -173,6 +173,7 @@ class OrderSheet extends StatelessWidget {
           ),
         ],
       );
+
```
Comment (tobischw) [inline example/lib/widgets/order_sheet.dart:176]: not needed?

**Review by tobischw — APPROVED** [review]

**Comment by pauljonescodes** [comment]
Is this getting published? https://pub.dev/packages/square_in_app_payments/versions
